README.md

```diff
@@ -134,23 +131,50 @@ new_drd.transitions # => ['self', 'edit', 'delete', 'deactivate', 'leviathan']
 For more examples and information on using Faraday with media-types that require specifying uniform-interface methods 
 and other protocol idioms when invoking transitions, see [Using Farscape]().
 
-## Bookmarks
-Following you nose is great, but if you have been there and you want to get back quickly, bookmarks are sweet 
-(particularly since Farscape can cache prior requests using rack middleware). A bookmark is simple hash that allows 
-easily reloading a prior state of a resources (of course things may have changed since you were there so you must 
-temper your assumptions).
+## Alternate Interface
+
+For developers more used to ActiveRecord syntax, Farscape resources also expose all transitions and attributes as Ruby methods. Safe (i.e. read) transitions are exposed verbatim.
 
 ```ruby
-drds = agent.enter(:drds)
+drd.leviathan # => Equivalent to drd.transitions['leviathan'].invoke
+```
 
-self_transition = drds.transitions['self']
-drds_bookmark = self_transition.bookmark
+Unsafe transitions have an exclamation point at the end.
 
-drds = agent.invoke(bookmark: drds_bookmark)
+```ruby
+drd.deactivate # => Raises NoMethodError
+
+drd.deactivate! # => Equivalent to drd = drd.transitions['deactivate'].invoke
+```
+
+Request parameters can be passed as a hash or as a block.
+
+```ruby
+# The following are all equivalent:
+
+drd = drds.create!(name: 'Pike')
+drd = drds.create! { |builder| builder.attributes = {name: 'Pike'} }
+drd = drds.transitions['create'].invoke{ |d| d.attributes = {name: 'Pike'} }
+```
+
+Attributes are read-only.
+
+```ruby
+drd.name # => "Pike"
+
+drd.name = 'Susan' # => Raises NoMethodError
+```
+
+If an attribute or transition's name conflicts with an existing method or reserved word, it will not be methodized and must be accessed through the hash interface.
+
+### Disabling the Alternate Interface
+If you're concerned about namespace collisions, or want to ensure that your code is highly flexible and explicit (albeit verbose), you may turn off the interface with the .safe method.
```

Comment:
I think my goal here is just to avoid a 3 second "wat" for someone who thinks
```
safe_representor.unsafe
safe_representor.name
```
is going to work out.  I think you might be right in that the clearest way to signal that would be that one additional line of code.  Oh while someone is in there, could we fix the references to transitions returning an array of strings? I think they should either be transitions.keys calls or indicate a hash of strings and TransitionAgents right?